apps/src/applab/applab.js

```diff
@@ -680,7 +694,7 @@ Applab.init = function(config) {
     // should never be present on such levels, however some levels do
     // have levelHtml stored due to a previous bug. HTML set by levelbuilder
     // is stored in startHtml, not levelHtml.
-    if (config.level.hideDesignMode) {
+    if (Applab.reduxStore.getState().isDesignModeHidden) {
```

Comment:
Made a comment about this elsewhere as well, but I think it should be a goal to directly access the store as little as possible.